doc/user/content/sql/functions/now_and_mz_now.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "now and mz_now functions"
+description: "In Materialize mz_now() acts the way now() does in most systems."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+In Materialize, `now()` doesn't represent the system time, as it does in most systems; it represents the time with timezone when the query was executed. It cannot be used when creating views.
+
+`mz_now()` comes closer to what `now()` typically indicates. It represents the logical time at which a query executes. Its typical uses are:
```

Comment:
```suggestion
In Materialize, `now()` returns the value of the system clock with timezone when the transaction began. It cannot be used when creating views.

By contrast, `mz_now()` returns the logical time at which the query was executed. This may be arbitrarily ahead of or behind the system clock.

For example, at 9pm, Materialize may choose to execute a query as of logical time 8:30pm, perhaps because data for 8:30–9pm has not yet arrived. In this scenario, `now()` would return 9pm, while `mz_now()` would return 8:30pm.

Its typical uses are:
```